src/neural/network_cudnn.cu

```diff
@@ -358,25 +357,25 @@ __global__ void batchNormForward_kernel(T *output, const T *input,
   output[index] = (T)el;
 }
 
-// each thread processes single element
+// Every thread processes single element.
```

Comment:
I would think that "each" is more appropriate than "every"?

(also "processes a single element")